deploy/helm/kuberhealthy/templates/service.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: {{ template "kuberhealthy.name" . }}
+  name: {{ template "kuberhealthy.name" . }}
+  {{- if or .Values.prometheus.enabled .Values.service.annotations }}
+  annotations:
+  {{- if .Values.prometheus.enabled }}
```

Comment:
I've set up prometheus to scrape the service in front of the pods here. Maybe remove the one from the deploy and keep this?